service/test/integration2_test.go

```diff
@@ -1709,7 +1709,7 @@ func TestAcceptMulticlient(t *testing.T) {
 			},
 		})
 		if err := server.Run(); err != nil {
-			t.Fatal(err)
+			panic(err)
```

Comment:
This is the only change that doesn't make immediate sense to me.

Reply:
I had the same change on my other branch. It seems that DeepSource complaints about this, since the error is related to the server run but not the test itself, thus it seems more appropriate to panic instead of fail the test.

Reply:
Ack, makes sense.